active/0000-service-workers.md

```diff
@@ -0,0 +1,48 @@
+- Start Date: 2018-03-13
+- RFC PR: (leave this empty)
+
+# Add a Service Worker with minimal asset caching to a default Ember CLI application
+## Summary
+This RFC introduces a Service Worker to the default app blueprint. It will only cache the most essential assets, such as `index.html`, and the vendor and app specific JS and CSS files.
+
+## Motivation
+A Service Worker allows you to cache assets and other resources in a  way that loading a page becomes reliable and fast, independent of what the network conditions are.
+
+Not all Ember applications are destined to become a Progressive Web App, but that doesn't mean that applications shouldn't leverage a simple Service Worker script to somewhat improve the reliability of loading it.
+
+## Detailed design
+### Registration
+A Service Worker script named `sw.js` should be compiled and placed at the root level of the build result. The Service Worker script should be registered with a registration script that is inlined in `index.html`. The registered Service Worker should have the `rootURL` that is configured in `config/environment.js` as scope.
+
+### Caching
+The default Service Worker configuration should only cache the following files:
+
+* `index.html`
```

Comment:
How will this default set be handled for non-standard apps? For example, if you are using prember, and not using SW, then this default gets turned on, it will troll you because what you want for prember is caching all the individual index.html files, and if it does the root one, you get it always trying to load the root page of your app.